### PR TITLE
Use .into() vir::Expr instances in some more places

### PR DESCRIPTION
**Pull request by dario23**

### CODE REVIEW

File: prusti-common/src/vir/ast/expr.rs [hunk 1]
```diff
@@ -339,6 +339,18 @@ impl Expr {
...
+        Expr::Const(Const::Int(i.into()), Position::default())
+    }
+
+    pub fn const_bigint<S: Into<String>>(s: S) -> Self {
+        Expr::Const(Const::BigInt(s.into()), Position::default())
+    }
+
+    pub fn const_bool(b: bool) -> Self {
+        Expr::Const(Const::Bool(b), Position::default())
+    }
```
Comment (Aurel300) [inline prusti-common/src/vir/ast/expr.rs]: Aren't there already `Into` implementations for (some of) these?

File: prusti-viper/src/encoder/encoder.rs [hunk 2]
```diff
@@ -1304,7 +1304,7 @@ impl<'v, 'tcx> Encoder<'v, 'tcx> {
                     ),
                     vir::Position::default(),
                 ),
-                box vir::Expr::Const(vir::Const::Bool(true), vir::Position::default()),
+                box vir!{ true },
```
Comment (Aurel300) [inline prusti-viper/src/encoder/encoder.rs]: Just `true.into()` should work.

File: prusti-common/src/vir/ast/expr.rs (hunk 1 above)
Comment (dario23) [reply prusti-common/src/vir/ast/expr.rs]: hm, yes; that makes the PR one commit shorter i guess :-)

File: prusti-common/src/vir/optimizations/functions/simplifier.rs [hunk 3]
```diff
@@ -7,6 +7,7 @@
 //! Function simplifier that simplifies expressions.
 
 use super::super::super::ast::{self, ExprFolder};
+use crate::vir;
```
Comment (Aurel300) [inline prusti-common/src/vir/optimizations/functions/simplifier.rs]: I guess this is no longer necessary.

**Pull request merged**